src/pybind/mgr/dashboard/tools.py

```diff
@@ -473,27 +617,27 @@ def stop(cls):
         logger.debug("notification queue stopped")
 
     @classmethod
-    def register(cls, func, types=None):
+    def register(cls, func, n_types=None):
```

Comment:
Can you put this renaming part into a separate commit?

Reply:
fixed